### ISSUE TITLE
Separate access to query description and query target segment

### ISSUE THREAD

**Issue by ppanopticon** [lucaro/DRES#66]
This is a follow-up task for #56.

There should be a dedicated API that can be used to access the _query target segment_ for a specific task run (e.g. the video we're looking for) so that we can access the target segment and the query description separately e.g. for KIS Textual Tasks, where we need the target segment only after the competition has finished. 

Based on our current experience, we have following design goals:

- The API returns the pre-rendered target segments for a given task.
- The API should require the ID of the **task run** (not the task) as parameter. This will prevent illegal access to the segment by exploiting the internal task IDs.
- Th API should only return a value if the task run in question has completed so as to prevent leakage of information relevant to the competition.
- If the task in question does not have a specific target segment (e.g. for AVS tasks) then the API should report an error.

Feel free to add more ideas.

**Comment by lucaro** [comment]
The API should also depend on some task id in order to avoid having to deal with browser caches.

**Comment by ppanopticon** [comment]
Maybe it could only depend on the task ID but internally restricted so that it can only be used to access the segment of a recently completed task. I guess we have a session based associated of caller / user to run in other parts of the stack already, don't we?

**Comment by lucaro** [comment]
Yes, such a restriction is possible. It further needs to differentiate on the type of task, such that the videos for textual tasks are only served once the task is no longer running.

**Comment by sauterl** [comment]
Addressed with #146  and following
